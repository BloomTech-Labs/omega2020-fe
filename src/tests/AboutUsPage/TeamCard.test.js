import React from 'react';
import { render } from '@testing-library/react';
import  { configure, shallow } from 'enzyme';
import '@testing-library/jest-dom/extend-expect';
import List from '../../components/AboutUsPage/TeamCard';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';

configure({ adapter: new Adapter() });

describe('the components', () => {

    let wrapper;
        beforeEach(() => { wrapper = shallow(<List />)});
        
        it('renders the snapshot', () => {
            const wrapper = renderer.create(<List />).toJSON();
            expect(wrapper).toMatchSnapshot();
        })
        
        it('check if the compnonent exist', () => {
        expect(wrapper.exists()).toBe(true);
        });

      
})

