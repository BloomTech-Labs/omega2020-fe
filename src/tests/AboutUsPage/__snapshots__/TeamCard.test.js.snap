// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the components renders the snapshot 1`] = `ReactWrapper {}`;
